Makefile

```
@@ -23,42 +22,38 @@ clean:
-	uv run pre-commit run --all-files
+	uv run --no-sync pre-commit run --all-files
 
 release-notes:
-	uv run scripts/generate_release_notes.py
+	uv run --no-sync scripts/generate_release_notes.py
 
 sbom:
-	uv sync --extra build --link-mode=copy
 	uv tool run --from cyclonedx-bom cyclonedx-py environment "$(uv python find)"
 
 setup:
```

checkmake (0.2.2)

[warning] Target body for "setup" exceeds allowed length of 5 (6).

(maxbodylength)